### ISSUE TITLE
Unsatisfactory results

### ISSUE THREAD

**Issue by kukukiki765** [AlonEirew/cross-doc-event-coref#10]
Hi, thank you for your work! But I seem to have some problems reproducing the model's performance. I followed the process you provided step by step, but I always couldn't achieve the optimal results given in the article despite trying several different sets of parameters. So I want to know what parameters can achieve the optimal results and how they differ from the default parameters
default:
   -h --help       Show this screen.
    --bs=<x>        Batch size [default: 32]
    --lr=<y>        Learning rate [default: 5e-4]
    --ratio=<z>     Ratio of positive:negative, were negative is the controlled list (ratio=-1 => no ratio) [default: -1]
    --itr=<k>       Number of iterations [default: 10]
    --cuda=<y>      True/False - Whether to use cuda device or not [default: True]
    --ft=<b1>       Fine-tune the LM or not [default: False]
    --wd=<t>        Adam optimizer Weight-decay [default: 0.01]
    --hidden=<w>    hidden layers size [default: 150]
    --dataset=<d>   wec/ecb - which dataset to generate for [default: wec]

**Comment by AlonEirew** [comment]
Hi @LinfanLiu01, I don't remember which parameters I've used, however I found a copy of the model, hopefully it's the one with the results reported in the paper, hope this helps you.
[download link](https://drive.google.com/file/d/1GX9hjIBK8mTk3Me58ZprAw9gc1oH-tnJ/view?usp=sharing)

**Comment by AlonEirew** [comment]
Hi @LinfanLiu01, I don't remember which parameters I've used, however did you try and reproduce using the model I've shared in huggingface model hub (link in the project readme)?

**Comment by kukukiki765** [comment]
Yes, I also use the weights you shared for the inference process and clustering, but strangely, the results obtained from this are even much lower than those given in the paper. I don't know where the problem comes from. Do I need to make adjustments in other aspects besides the parameters you set in the command
![1](https://github.com/user-attachments/assets/51a50e0c-724c-4847-899d-bc81f6a1d76a)
![2](https://github.com/user-attachments/assets/00bf2211-61bf-40b8-a01a-2607e3e276c1)

**Comment by AlonEirew** [comment]
Given these results, I assume you have a bug or an error in the code or in following the training procedure..

**Comment by kukukiki765** [comment]
It is very likely. I will re-examine my program and execution process
